### DISCUSSION TITLE
Windows 11 icon for Windows 10 taskbar style

### DISCUSSION THREAD

**Discussion by farsdewibs0n** [valinet/ExplorerPatcher#2687] [Q&A]
Wonder if there's a way to change the icon style to Windows 11 when using the Windows 10 taskbar style
![gambar](https://github.com/valinet/ExplorerPatcher/assets/13050356/3bfca370-d988-4008-a63c-9001486338af)

**Comment by pyrates999** [comment]
I'd say this as a no.
